src/runtime_src/core/pcie/driver/linux/xocl/subdev/icap.c

```diff
@@ -697,7 +697,7 @@ static int icap_xclbin_validate_clock_req(struct platform_device *pdev,
 	struct icap *icap = platform_get_drvdata(pdev);
 	uint32_t slot_id = 0;
 	struct islot_info *islot = NULL;
-	int err;
```

Comment:
@saifuddin-xilinx do you think the err=0 is good or err=-EINVAL should be set by default for this special case?

Reply:
err=0 is fine.